### ISSUE TITLE
Avoid false-positive for unknown property (S4654) inside :export selector

### ISSUE THREAD

**Issue by micmarc** [SonarSource/sonar-css#258]
I am experiencing an issue similar to #149 and #152, and I believe that PR #171 has not properly fixed the issue. I have something like the following in `_variables.scss`: 
```
$light-blue-10: #c0e7f3;
$light-blue-40: #4bc7e7;
$teal: #23B8A7;
:export {
    lightBlue10: $light-blue-10;
    lightBlue40: $light-blue-40;
    teal: $teal;
}
```
This causes a blocker violation for each exported property, e.g.:
```
Unexpected unknown property "lightBlue10"
```

The original fix added [a pseudo-selector exclusion for "export" and "import"](https://github.com/SonarSource/sonar-css/pull/171/files#diff-d447676eaf4c8790e45ed6ae609894caR42), but I believe we also need a selector exclusion for these within the `property-no-unknown` rule as indicated in the [Stylelint docs](https://stylelint.io/user-guide/rules/property-no-unknown#ignoreselectors-regex-regex-string). 

The [exclusion of specific example property values `exportedKey` and `localAlias`](https://github.com/SonarSource/sonar-css/pull/171/files#diff-0fbf01c6609b6582d0d4d15b7602df4aR42) in the original PR, while not hurting anything, doesn't seem to be helping either. I believe that removing these exclusions while also adding the property selector exclusion will lead to the same result but be more flexible in terms of the actual property names.

**Comment by vilchik-elena** [comment]
Indeed we can rely on `ignoreSelectors` parameter of `property-no-unknown`.  
`exportedKey` and `localAlias` should be removed as these are just examples from the documentation (🤦)

**Comment by micmarc** [comment]
@vilchik-elena Would it be OK if I raised a PR for this?
